### DISCUSSION TITLE
I have access for the Hindi translation of OpenTTD but I need help translating.

### DISCUSSION THREAD

**Discussion by NisheshTyagi** [OpenTTD/OpenTTD#12111] [Q&A] [answered]
I can't seem to figure out how to translate properly. There are various confusing string and I don't know how to properly translate them like:
_{BLACK}Build or generate industries_
Should I keep the "{BLACK}" part and translate everything else?
What is
_"Plural syntax: {P [index] "1" "other" }"_?
Is there any documentation to understand how to handle these types of string and what is the syntax of the translations files?
It would help me a lot.

**Comment by rubidium42** [comment, marked as answer]
There is https://translator.openttd.org/static/docs/strings.html which should explain most.

**Reply by NisheshTyagi** [reply]
Oh, thanks!
